05LessonFive/Checkpoint1/index.html

```diff
@@ -1,10 +1,39 @@
 <!DOCTYPE html>
+
 <html>
+
 <head>
-  <title>Landscape Design</title>
+
+  <title>127.0.0.1:59695/evaluation.html</title>
+
   <link rel="stylesheet" href="css/style.css">
+
 </head>
+
 <body>
 
+  <main>
+
+    <Section>
```

Comment:
great work on this! minor thing, try not to capitalize the first letter in ANY element as it's just something devs do.